CHANGELOG.md

```diff
@@ -1,55 +0,0 @@
-<!---
```

Comment:
Removed the changelog because prefer to use GitHub releases for release documentation.